terraform/modules/firewall/variables.tf

```diff
@@ -28,3 +28,8 @@ variable "net_tags" {
   description = "tags for the firewall"
   type        = "list"
 }
+
+variable "execution_id" {
+  description = "The random number for each execution"
```

Comment:
```suggestion
  description = "A random string used as as suffix for resources."
```

Reply:
Shouldn't we be using one random string for the entire project?  Would make deleting resources A LOT simpler.

Reply:
That execution_id is used across the entire project.